pkg/state/state.go

```diff
@@ -183,6 +183,11 @@ type RepositorySpec struct {
 type ReleaseSpec struct {
 	// Chart is the name of the chart being installed to create this release
 	Chart string `yaml:"chart,omitempty"`
+
+	// ChartPath is the downloaded and modified version of the remote Chart specified by the Chart field.
+	// This field is empty when the release is going to use the remote chart as-is, without any modifications(e.g. chartify).
+	ChartPath string `yaml:"chartPath,omitempty"`
+
 	// Directory is an alias to Chart which may be of more fit when you want to use a local/remote directory containing
 	// K8s manifests or Kustomization as a chart
 	Directory string `yaml:"directory,omitempty"`
```

Comment:
i'm wondering about this field, can we use it as ChartPath?

Reply:
It's quite close but I still think a new field is nicer!
That's because `directory` is meant to be used by users, whereas `chartPath` is mainly for internal-use by helmfile. (Note that I added `yaml:"chartPath,omitempty"` so that it's included in `helmfile build` output which I thought might be helpful for debugging)